benchmarks/python/test_rope.py

```diff
@@ -82,5 +84,5 @@ def fwd_call(inp):
     # NOTE: the iobytes is computed based on how thunder autograd worked. So this is just
     # a reference point for torchcompile and eager executor for comparison.
     run_benchmark(
-        benchmark, unary_bwd_torch, [output, grad(), fwd_inputs()], iobytes=iobytes()
+        benchmark, unary_bwd_torch, [output, grad(), *fwd_inputs], iobytes=iobytes()
```

Comment:
The reason I need to expand `fwd_inputs` is to have it work with the infra code where we clear `grad` from `torch.Tensor` inputs. Since `fwd_inputs` is a sequence of torch.Tensor.

An alternative is to flatten `inputs` [here](https://github.com/NVIDIA/Fuser/blob/8ea30c7f796452bd696fbda79327d8851823df2b/benchmarks/python/core.py#L334-L336) instead.